pkgs/common-updater/scripts/list-directory-versions

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import argparse
+import requests
+import os
+import subprocess
+import json
+import re
+from bs4 import BeautifulSoup
+
+parser = argparse.ArgumentParser(
+    description="Get all available versions listed for a package in a site."
+)
+
+parser.add_argument(
+    "--pname",
+    default=os.environ.get("UPDATE_NIX_PNAME"),
+    required="UPDATE_NIX_PNAME" not in os.environ,
+    help="name of the package",
+)
+parser.add_argument(
+    "--attr-path",
+    default=os.environ.get("UPDATE_NIX_ATTR_PATH"),
```

Comment:
```suggestion
    default=os.environ.get("UPDATE_NIX_ATTR_PATH"),
    required="UPDATE_NIX_ATTR_PATH" not in os.environ,
```

Reply:
I think it is reasonable to have `attr_path` defaulting to `pname` in case neither the command line argument nor the environment variable is used. 

Line 35:
``` python
    attr_path = args.attr_path or pname
```

Reply:
Sorry, did not notice that. LGTM, then.